tap/go.mod

```diff
@@ -3,10 +3,15 @@ module github.com/up9inc/mizu/tap
 go 1.16
 
 require (
+	github.com/bradleyfalzon/tlsx v0.0.0-20170624122154-28fd0e59bac4 // indirect
 	github.com/google/gopacket v1.1.19
-	github.com/google/martian v2.1.0+incompatible
-	github.com/orcaman/concurrent-map v0.0.0-20210106121528-16402b402231
 	github.com/romana/rlog v0.0.0-20171115192701-f018bc92e7d7
-	golang.org/x/net v0.0.0-20210421230115-4e50805a0758
-	github.com/bradleyfalzon/tlsx v0.0.0-20170624122154-28fd0e59bac4
+	github.com/up9inc/mizu/tap/api v0.0.0
+	golang.org/x/net v0.0.0-20210224082022-3d97a244fca7
+	golang.org/x/sys v0.0.0-20210225134936-a50acf3fe073
+	golang.org/x/term v0.0.0-20210220032956-6a3ed077a48d
+	golang.org/x/text v0.3.5
+	golang.org/x/tools v0.0.0-20210106214847-113979e3529a
```

Comment:
@nimrod-up9 Go plugins (HTTP, AMQP and Kafka extensions) fail to load if there is a shared package between a driver program and the Go plugin that has different versions.

In our case, `golang.org/x/net` is shared between the driver program and the protocol extensions. The other listed packages are the dependencies of `golang.org/x/net`. Those lines are there to remind us lock the versions of those dependencies or upgrade them simultaneously.

Reply:
hmmm... There has to be a better way to manage the dependencies. We shouldn't manage them manually, it will break easily.
We can merge like this and fix it later. Let me know if you have a better idea.

Reply:
@nimrod-up9 yes, those lines can be deleted by a later commit. At least with this PR, it will be in the files history as an info.